### ISSUE TITLE
Error when uploading empty alignment file

### ISSUE THREAD

**Issue by stenglein-lab** [stenglein-lab/TaxAssessor#32]
If you upload an empty alignment file you get this error:

Error Uploading File: Nextera_isolate3_spade_contigs.fa.bn_nt
(1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1")

Perhaps check that file is non-empty before uploading. 

Do other file format sanity checks exist?

**Comment by josephtallison** [comment]
Agreed.  A quick check that will send a more intelligible response would be good there!

**Comment by josephtallison** [comment]
Agreed.  A quick check that will send a more intelligible response would be good there!
